src/sql/workbench/services/contextualization/common/serverContextualizationService.ts

```diff
@@ -105,9 +123,13 @@ export class ServerContextualizationService extends Disposable implements IServe
 		const providerName = this._connectionManagementService.getProviderIdFromUri(ownerUri);
 		const handler = this.getProvider(providerName);
 		if (handler) {
+			this._logService.info(`Getting server contextualization for ${ownerUri}`);
+
 			return await handler.getServerContextualization(ownerUri);
 		}
 		else {
+			this._logService.info(`No server contextualization provider found for ${ownerUri}`);
```

Comment:
Is it fine to include owner URI in the log? If it shouldn't be included, I'll remove it.

Reply:
Yup, logs are local to the user machine so stuff like file paths are fine to include (and very useful when debugging issues)